DSP_Plugins.GalacticScale/Scripts/PatchStarSystemGeneration/Rework/ReworkCreatePlanet.cs

```diff
@@ -111,13 +111,13 @@ public static class ReworkPlanetGen {
                 planetData.sunDistance = planetData.orbitRadius;
 
                 //Tidal Lock Management
-                if (randomNumber13 < Patch.ChanceTidalLock.Value) {
+                if (randomNumber12 < Patch.ChanceTidalLock.Value) {
```

Comment:
how is that fixing a bug ? 

The goal here was to have a planet either in TidalLocked or TidalLocked2 or etc ... 

With 1 draw of number. so how is that a bug ?